repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/files/FileHelper.java

```diff
@@ -0,0 +1,170 @@
+package fr.inria.spirals.repairnator.process.files;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
+import fr.inria.spirals.repairnator.Utils;
+import fr.inria.spirals.repairnator.process.step.AbstractStep;
+
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Created by Tellström on 05/07/18.
+ */
+
+public class FileHelper {
+	
+	private Logger getLogger() {
+        return LoggerFactory.getLogger(this.getClass());
+    }
```

Comment:
The indentation looks like messed up here. We have some checkstyle in this repo, and this indentation will break Travis build because of the checkstyle. Please, check the indentation of all the changes.

Reply:
This is funny. One year and a half ago the same thing happened to me, and then Simon put the checkstyle in the repo =X

Which IDE are you using?

Reply:
Redid it anyway, unsure if anything changed.

Reply:
It changed, but it's still not correct. You should config your IDE to use 4 spaces instead of tabs for the indentation.